src/components/Review.jsx

```diff
@@ -0,0 +1,37 @@
+import { useParams } from "react-router-dom";
+import { useState, useEffect } from "react";
+import { fetchReviewByID } from "../api";
+
+const Review = () => {
+    const {review_id} = useParams();
+    const [review, setReview] = useState({});
+
+    useEffect(() => {
+        fetchReviewByID(review_id)
+        .then((data) => {
+            setReview(data);
+        })
+    })
+    
+    return <section>    
```

Comment:
Use conditional rendering here to let the user know, that the review is loading.